src/Migrate/custom/New-AzMigrateServerReplication.ps1

```diff
@@ -119,6 +127,36 @@ function New-AzMigrateServerReplication {
         # Specifies the Availability Zone to be used for VM creation.
         ${TargetAvailabilityZone},
 
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetVmtags]
+        # Specifies the tag to be used for VM creation.
+        ${VMTag},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetNicTags]
+        # Specifies the tag to be used for Nic creation.
+        ${NicTag},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputTargetDiskTags]
+        # Specifies the tag to be used for Disk creation.
+        ${DiskTag},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20210210.IVMwareCbtEnableMigrationInputSeedDiskTags]
+        # Specifies the tag to be used for Seed Disk creation.
+        ${SeedDiskTag},
+
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [System.Collections.Hashtable]
+        # Specifies the tag to be used for Resource creation.
+        ${Tag},
```

Comment:
What tag is this resource for? VM, NIC, seed disk, target disk, that's all right?
        /// <summary>
        /// Gets or sets the target VM tags.
        /// </summary>
        [AsrMaxLength(MaxTagCountForVM)]
        [JsonProperty(PropertyName = "targetVmTags")]
        public Dictionary<string, string> TargetVmTags { get; set; }

        /// <summary>
        /// Gets or sets the tags for the seed disks.
        /// </summary>
        [AsrMaxLength(MaxTagCountForDisk - 1)]
        [JsonProperty(PropertyName = "seedDiskTags")]
        public Dictionary<string, string> SeedDiskTags { get; set; }

        /// <summary>
        /// Gets or sets the tags for the target disks.
        /// </summary>
        [AsrMaxLength(MaxTagCountForDisk - 1)]
        [JsonProperty(PropertyName = "targetDiskTags")]
        public Dictionary<string, string> TargetDiskTags { get; set; }

        /// <summary>
        /// Gets or sets the tags for the target NICs.
        /// </summary>
        [AsrMaxLength(MaxTagCountForNic)]
        [JsonProperty(PropertyName = "targetNicTags")]
        public Dictionary<string, string> TargetNicTags { get; set; }

Reply:
Yes, this tag sets all resources(VM, NIC, Disk) tag values.

Reply:
Yes